airflow/jobs.py

```diff
@@ -789,6 +795,10 @@ def _process_task_instances(self, dag, queue):
                                    .format(run.execution_date))
                 continue
 
+            # skip backfill dagruns for now as long as they are not really scheduled
+            if "backfill" in run.run_id:
```

Comment:
To make this contract more solid, maybe do something like `DagRun.is_backfill()`?